modules/uolBidAdapter.js

```diff
@@ -0,0 +1,202 @@
+import * as utils from 'src/utils';
+// import {config} from 'src/config';
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'uol';
+const ENDPOINT_URL = 'https://prebid.adilligo.com/v1/prebid.json';
+const UOL_LOG_HEADER = 'UOL Bidder Error: '
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: ['banner'], // not required since it is default
```

Comment:
Could import/use the defined mediaTypes from `src/mediaTypes`.  Eg `import { BANNER } from 'src/mediaTypes';`  This way you can ensure the values are read correctly.